src/ui_nbgl.c

```diff
@@ -171,7 +179,7 @@ static void _display_confirmation(void)
 		.progressIndicator = true,
 		.navWithTap.backButton = false,
 		.navWithTap.nextPageText = NULL,
-		.navWithTap.quitText = "Reject",
```

Comment:
Is this exclusively used for transactions? It seems the only place that uses it is `display_confirmation`, but that is called in many other places not related to transaction signing.